frontend/pages/ManageControlsPage/ManageControlsPage.tsx

```diff
@@ -73,6 +76,15 @@ const ManageControlsPage = ({
     includeNoTeam: true,
   });
```

Comment:
@lukeheath,
I think you can accomplish what you're trying to do by using optional `permittedAccessByTeamRole` property of the hook and pulling `userTeams` from the hook instead of `AppContext`. 

```
permittedAccessByTeamRole: {
      admin: true,
      maintainer: true,
      observer: false,
    }
```